include/hpp/core/path-optimization/quadratic-program.hh

```diff
@@ -62,31 +62,68 @@ struct QuadraticProgram {
   typedef Eigen::JacobiSVD<matrix_t> Decomposition_t;
   typedef Eigen::LLT<matrix_t, Eigen::Lower> LLT_t;
 
-  QuadraticProgram(size_type inputSize)
+  /// Constructor
+  /// \param inputSize dimension of the space on which the quadratic cost is
+  ///                  defined,
+  /// \param useProxqp whether to use proxqp instead of eiquadprog_2011 as
+  ///                  the internal solver. This parameter will soon be removed
+  ///                  and proxqp be the internal solver.
+  QuadraticProgram(size_type inputSize, bool useProxqp = true)
       : H(inputSize, inputSize),
         b(inputSize),
         dec(inputSize, inputSize, Eigen::ComputeThinU | Eigen::ComputeThinV),
-        xStar(inputSize) {
+        xStar(inputSize),
+        accuracy_(1e-4),
+        useProxqp_(useProxqp) {
     H.setZero();
     b.setZero();
     bIsZero = true;
   }
 
-  QuadraticProgram(const QuadraticProgram& QP, const LinearConstraint& lc)
+  /// Constructor
+  /// \param inputSize dimension of the space on which the quadratic cost is
+  ///                  defined,
+  /// \param lc        linear equality constraint,
+  /// \param useProxqp whether to use proxqp instead of eiquadprog_2011 as
+  ///                  the internal solver. This parameter will soon be removed
+  ///                  and proxqp be the internal solver.
+  QuadraticProgram(const QuadraticProgram& QP, const LinearConstraint& lc,
+                   bool useProxqp = true)
       : H(lc.PK.cols(), lc.PK.cols()),
         b(lc.PK.cols()),
         bIsZero(false),
         dec(lc.PK.cols(), lc.PK.cols(),
             Eigen::ComputeThinU | Eigen::ComputeThinV),
-        xStar(lc.PK.cols()) {
+        xStar(lc.PK.cols()),
+        accuracy_(1e-4),
```

Comment:
Any reason to choose 1e-4? It seems to be a large value to me.

Reply:
Taking a too small value makes proxqp reach the maximal number of iterations often. I actually did not try other values between 1e-9 and 1e-4. It has no influence on the continuity of trajectories, only on the cost of the solution returned by the QP solver as well as on the inequality constraints. We may change this value after comprehensive tests.

Reply:
Ok. Let's stick to this value.